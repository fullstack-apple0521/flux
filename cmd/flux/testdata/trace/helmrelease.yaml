---
apiVersion: v1
kind: Namespace
metadata:
  name: flux-system
---
apiVersion: v1
kind: Namespace
metadata:
  name: podinfo
---
apiVersion: helm.toolkit.fluxcd.io/v2beta1
kind: HelmRelease
metadata:
  labels:
    kustomize.toolkit.fluxcd.io/name: infrastructure
    kustomize.toolkit.fluxcd.io/namespace: flux-system
  name: podinfo
  namespace: podinfo
spec:
  chart:
    spec:
      chart: podinfo
      sourceRef:
        kind: HelmRepository
        name: podinfo
        namespace: flux-system
  interval: 5m
status:
  conditions:
  - lastTransitionTime: "2021-07-16T15:42:20Z"
    message: Release reconciliation succeeded
    reason: ReconciliationSucceeded
    status: "True"
    type: Ready
  helmChart: flux-system/podinfo-podinfo
  lastAppliedRevision: 6.0.0
  lastAttemptedRevision: 6.0.0
  lastAttemptedValuesChecksum: c31db75d05b7515eba2eef47bd71038c74b2e531
---
apiVersion: kustomize.toolkit.fluxcd.io/v1beta1
kind: Kustomization
metadata:
  name: infrastructure
  namespace: flux-system
spec:
  path: ./infrastructure
  sourceRef:
    kind: GitRepository
    name: flux-system
  validation: client
  interval: 5m
  prune: false
status:
  conditions:
  - lastTransitionTime: "2021-08-01T04:52:56Z"
    message: 'Applied revision: main/696f056df216eea4f9401adbee0ff744d4df390f'
    reason: ReconciliationSucceeded
    status: "True"
    type: Ready
  lastAppliedRevision: main/696f056df216eea4f9401adbee0ff744d4df390f
---
apiVersion: source.toolkit.fluxcd.io/v1beta1
kind: GitRepository
metadata:
  labels:
    kustomize.toolkit.fluxcd.io/name: flux-system
    kustomize.toolkit.fluxcd.io/namespace: flux-system
  name: flux-system
  namespace: flux-system
spec:
  gitImplementation: go-git
  ref:
    branch: main
  secretRef:
    name: flux-system
  url: ssh://git@github.com/example/repo
  interval: 5m
status:
  artifact:
    lastUpdateTime: "2021-08-01T04:28:42Z"
    revision: main/696f056df216eea4f9401adbee0ff744d4df390f
    path: "example"
    url: "example"
  conditions:
  - lastTransitionTime: "2021-07-20T00:48:16Z"
    message: 'Fetched revision: main/696f056df216eea4f9401adbee0ff744d4df390f'
    reason: GitOperationSucceed
    status: "True"
    type: Ready
