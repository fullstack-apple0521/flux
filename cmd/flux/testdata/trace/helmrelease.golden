
Object:        HelmRelease/podinfo
Namespace:     {{ .ns }}
Status:        Managed by Flux
---
Kustomization: infrastructure
Namespace:     {{ .fluxns }}
Path:          ./infrastructure
Revision:      main/696f056df216eea4f9401adbee0ff744d4df390f
Status:        Last reconciled at {{ .kustomizationLastReconcile }}
Message:       Applied revision: main/696f056df216eea4f9401adbee0ff744d4df390f
---
GitRepository: flux-system
Namespace:     {{ .fluxns }}
URL:           ssh://git@github.com/example/repo
Branch:        main
Revision:      main/696f056df216eea4f9401adbee0ff744d4df390f
Status:        Last reconciled at {{ .gitRepositoryLastReconcile }}
Message:       Fetched revision: main/696f056df216eea4f9401adbee0ff744d4df390f
